ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -185,6 +185,7 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
             }
```

Comment:
It's a bit off screen, but I think we're trying to set the accessibility label of the button without a title through `.accessibility(identifier: "Accessibility.Dismiss.Label")`, but considering that VO reads the X as "dismiss-20x20", I don't think that's working.

Reply:
why are we setting identifier? shouldn't it be label? https://developer.apple.com/documentation/swiftui/view/accessibilitylabel(_:)-1d7jv

Reply:
I assume the hope was that passing in the identifier would pull the correct string from our string files. Updated it to `.accessibilityLabel`.